deploy/docker/sysrepo-netopeer2/supervisord.conf

```diff
@@ -0,0 +1,22 @@
+[supervisord]
+nodaemon=true
+logfile=/var/log/supervisord.log
+loglevel=debug
+
+[program:sysrepod]
+command=/usr/local/bin/sysrepod -d
+autorestart=true
+redirect_stderr=true
+priority=1
+
+[program:sysrepo-plugind]
+command=/usr/local/bin/sysrepo-plugind -d
+autorestart=true
+redirect_stderr=true
+priority=2
```

Comment:
Probably should be removed, if the user will use any plugins, they can start the daemon themselves.